elementary/monitor/dbt_project/models/enriched_exposures/enriched_exposures.sql

```diff
@@ -5,15 +5,33 @@ config(
   unique_key='unique_id',
 )
 }}
+{% set dbt_exposures_relation = ref('elementary', 'dbt_exposures') %}
+{% set raw_queries_column_exists = elementary.column_exists_in_relation(dbt_exposures_relation, 'raw_queries') %}
+{% set columns_without_raw_queries = [
+    'unique_id',
+    'name',
+    'maturity',
+    'type',
+    'owner_email',
+    'owner_name',
+    'url',
+    'depends_on_macros',
+    'depends_on_nodes',
+    'description',
+    'tags',
+    'meta',
+    'package_name',
+    'original_path',
+    'path',
+    'generated_at',
+    'metadata_hash',
+    'label'
+] %}
 
-with dbt_exposures as (
-select * from {{ ref('dbt_exposures') }}
-),
-
-elementary_exposures as (
-select * from {{ ref('elementary_cli', 'elementary_exposures') }}
-)
-
-{# Union without duplicates where elementary_exposures has prio #}
-select * from dbt_exposures where dbt_exposures.unique_id not in (select elementary_exposures.unique_id from elementary_exposures)
-{{ elementary.sql_union_distinct() }} select * from elementary_exposures
+select
+  {% for column in columns %}
```

Comment:
Should it be columns_without_raw_queries instead of columns? also personally I'd prefer to put the list of columns explicitly here without the jinja for loop. I know it's not the most "engineering" way but in dbt they have this doc about best practices where they say that in SQL sometimes being clear (less jinja) is more important than DRY (do not repeat yourself) as the SQL becomes much clearer this way. Not critical, just worth mentioning this.
https://docs.getdbt.com/docs/build/jinja-macros#favor-readability-over--ness